notebooks/00_quick_start/wide_deep_model_movielens.ipynb

```diff
@@ -0,0 +1,1388 @@
+{
```

Comment:
<p>Maybe it would be worthwhile to add a small paragraph describing what is Wide&amp;Deep at a high level</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>As I'm new, a couple of questions:</p><p>Can we refer to it only as AzureML? </p><p>It had been mentioned to me that AML should be avoided as a reference? </p><p>Please inform me so I follow the convention going forward. </p><p>thanks!</p><p><br></p>

Reply:
That makes sense. The standard way we would take this action is to create an issue and then a person assigns himself or herself to achieve the tasks. Beth, would you be willing to take this task?

Reply:
@miguelgfierro Good point. Will add.
@bethz That is interesting... I heard no one says 'AzureML' but all say AML. I referred AzureML along with AML on the notebook only because the package name is `azureml`. But if what you say is true, we should definitely follow that. Can you confirm which is the right way to refer?

Reply:
<p><a href="https://github.com/miguelgfierro" target="_blank">@miguelgfierro</a>  <a href="https://github.com/loomlike" target="_blank">@loomlike</a>  Yes, I will confirm which is the right way to refer. I didn't have privs last week but I'll try to create the issue in GitHub now as I think that was fixed. </p>

Reply:
<p>I was able to successfully create an <a href="https://github.com/Microsoft/Recommenders/issues/483" target="_blank">issue </a> :-) </p><p><br></p>

Reply:
Changed to AzureML. For scripts, kept aml*